### DISCUSSION TITLE
NXP RT10XX support roadmap

### DISCUSSION THREAD

**Discussion by kesmik** [zephyrproject-rtos/zephyr#40921] [Q&A] [answered]
Hello,

I may be too new to NXP and Zephyr but I am very curious if there is a roadmap of ``NXP RT10XX`` series devices support in Zephyr platform.
The question raised then I have tried to check if SLEEP is supported and did not noticed that ``RT10XX`` has it implemented in sources.

I may missed something but if you have features which are planned to be implemented but not implemented yet please share. If there is some ways to find such information differently please let me know  I will search for it.

Best regards,
Kęstutis

**Comment by DerekSnell** [comment, marked as answer]
Hi @kesmik ,

NXP is not yet publishing our roadmap plans.  But our team is progressing on adding Power Manager support for the RT1064.  We plan to have this feature merged in time for Zephyr's v3.0 release.  And this will include support for the RT1064's Sleep mode.

Best regards

**Reply by kesmik** [reply]
Thank you, @DerekSnell, I think I got the answer.

I need to check more features which are missing and are needed for me (if there is any) to ask more direct, because currently I have checked only few.

BR,
Kęstutis
